_includes/footer.html

```diff
@@ -0,0 +1,62 @@
+<div class="mysoc-footer" role="contentinfo">
+    <div class="container">
+        <div class="row">
+
+            <div class="col-sm-5">
+                <h2 class="mysoc-footer__site-name">FixMyStreet Platform</h2>
+                <div class="mysoc-footer__site-description">
+                    <p>Open source report-mapping software that can be deployed anywhere in the world. Most commonly used for reporting street issues to councils, but flexible enough to fit any project that matches geographical points to email addresses.</p>
+                </div>
+            </div>
+
+            <div class="col-sm-4">
+                <nav class="mysoc-footer__links">
+                    <ul>
+                        <li role="presentation"><a href="{{ site.baseurl }}install/">Install</a></li>
+                        <li role="presentation"><a href="{{ site.baseurl }}overview/">Documentation</a></li>
+                        <li role="presentation"><a href="https://github.com/mysociety/fixmystreet/blob/master/LICENSE.txt">Licence info</a></li>
+                    </ul>
+                    <ul>
+                        <li role="presentation"><a href="{{ site.baseurl }}community/">Mailing list</a></li>
+                        <li role="presentation"><a href="{{ site.baseurl }}community/">IRC</a></li>
+                    </ul>
+                </nav>
+            </div>
+
+            <div class="col-sm-3">
+                <div class="mysoc-footer__donate">
+                    <p>Your donations keep this site and others like it running</p>
+                    <a href="https://www.mysociety.org/donate" class="mysoc-footer__donate__button">Donate now</a>
+                </div>
+            </div>
+
+        </div>
+        <hr class="mysoc-footer__divider" role="presentation">
+        <div class="row">
+
+            <div class="col-sm-5">
+                <div class="mysoc-footer__orgs">
+                    <p class="mysoc-footer__org">
+                        Built by
+                        <a href="https://www.mysociety.org" class="mysoc-footer__org__logo mysoc-footer__org__logo--mysociety">mySociety</a>
+                    </p>
+                </div>
+            </div>
+
+            <div class="col-sm-4">
+                <div class="mysoc-footer__legal">
+                    <p>mySociety Limited (05798215) is a project of UK Citizens Online Democracy (UKCOD) a registered charity in England and Wales, charity number 1076346, company number 03277032.</p>
+                </div>
+            </div>
+
+            <div class="col-sm-3">
+                <ul class="mysoc-footer__badges">
+                    <li role="presentation"><a href="https://github.com/mysociety/fixmystreet" class="mysoc-footer__badge mysoc-footer__badge--github">Github</a></li>
+                    <li role="presentation"><a href="https://www.twitter.com/mysociety" class="mysoc-footer__badge mysoc-footer__badge--twitter">Twitter</a></li>
```

Comment:
I wonder if fixmystreet is better twitter account, though that's about UK site, hmm.